[rospy.client][INFO] 2020-10-29 22:45:33,508: init_node, name[/rostful_server], pid[3589]
[xmlrpc][INFO] 2020-10-29 22:45:33,509: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-29 22:45:33,509: Started XML-RPC server [http://freight1207:58342/]
[rospy.init][INFO] 2020-10-29 22:45:33,509: ROS Slave URI: [http://freight1207:58342/]
[rospy.impl.masterslave][INFO] 2020-10-29 22:45:33,509: _ready: http://freight1207:58342/
[xmlrpc][INFO] 2020-10-29 22:45:33,510: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-29 22:45:33,510: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-29 22:45:33,609: registered with master
[rospy.rosout][INFO] 2020-10-29 22:45:33,609: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-29 22:45:33,611: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-29 22:45:33,612: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-29 22:45:33,616: rostful_server: Delaying the start 1 seconds
[rospy.internal][INFO] 2020-10-29 22:45:33,885: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2020-10-29 22:45:34,664: rostful_server: Started server on port 8080
[rospy.internal][INFO] 2020-10-29 22:45:34,675: topic[/current/pose] adding connection to [http://freight1207:46457/], count 0
[rospy.internal][INFO] 2020-10-29 22:45:34,677: topic[/current/pose] adding connection to [http://freight1207:58342/], count 1
[rospy.internal][INFO] 2020-10-29 22:45:34,678: topic[/remote_control] adding connection to [http://freight1207:58342/], count 0
[rospy.internal][INFO] 2020-10-29 22:45:34,679: topic[/current/pose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 22:45:34,680: topic[/enable_software_runstop] adding connection to [http://freight1207:58342/], count 0
[rospy.internal][INFO] 2020-10-29 22:45:34,681: topic[/local_control/cancel] adding connection to [http://freight1207:58342/], count 0
[rospy.internal][INFO] 2020-10-29 22:45:34,681: topic[/remote_control] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 22:45:34,682: topic[/local_control/initialpose] adding connection to [http://freight1207:58342/], count 0
[rospy.internal][INFO] 2020-10-29 22:45:34,683: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 22:45:34,683: topic[/local_control/cancel] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 22:45:34,684: topic[/local_control/goto] adding connection to [http://freight1207:58342/], count 0
[rospy.internal][INFO] 2020-10-29 22:45:34,684: topic[/local_control/initialpose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 22:45:34,684: topic[/local_control/goto] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 22:45:34,730: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
[rospy.internal][INFO] 2020-10-29 22:45:34,887: topic[/local_control/initialpose] adding connection to [/local_control_initialpose], count 1
[rospy.internal][INFO] 2020-10-29 22:45:34,888: topic[/local_control/cancel] adding connection to [/local_control_navigation], count 1
[rospy.internal][INFO] 2020-10-29 22:45:34,888: topic[/local_control/goto] adding connection to [/local_control_navigation], count 1
[rospy.internal][INFO] 2020-10-29 22:45:34,888: topic[/remote_control] adding connection to [/remote_control], count 1
[rosout][WARNING] 2020-10-29 22:45:47,352: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.877745524438
    locationY: 0.482889528928
    orientation: 0.47
  - 
    locationX: -0.0113198089107
    locationY: 2.41521978606
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:46:32,967: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.877745524438
    locationY: 0.482889528928
    orientation: 0.9
  - 
    locationX: -0.0113198089107
    locationY: 2.41521978606
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:46:37,794: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.877745524438
    locationY: 0.482889528928
    orientation: 0.9
  - 
    locationX: -0.0113198089107
    locationY: 2.41521978606
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:47:12,755: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.877745524438
    locationY: 0.482889528928
    orientation: 0.9
  - 
    locationX: -0.0113198089107
    locationY: 2.41521978606
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:47:19,565: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.877745524438
    locationY: 0.482889528928
    orientation: 0.9
  - 
    locationX: -0.0113198089107
    locationY: 2.41521978606
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:48:36,438: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.877745524438
    locationY: 0.482889528928
    orientation: 0.9
  - 
    locationX: -0.0113198089107
    locationY: 2.41521978606
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:49:33,395: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:49:36,117: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:49:41,228: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:53:35,842: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:53:42,418: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:57:05,827: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:57:19,437: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:57:20,544: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:57:21,567: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 22:57:25,572: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][INFO] 2020-10-29 22:58:32,969: rostful_server: Shutting down the server
[rospy.core][INFO] 2020-10-29 22:58:32,969: signal_shutdown [Closing]
[rospy.internal][INFO] 2020-10-29 22:58:32,973: topic[/enable_software_runstop] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 22:58:32,973: topic[/enable_software_runstop] removing connection to /software_runstop
[rospy.internal][INFO] 2020-10-29 22:58:32,973: topic[/local_control/goto] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 22:58:32,973: topic[/local_control/goto] removing connection to /local_control_navigation
[rospy.internal][INFO] 2020-10-29 22:58:32,974: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-10-29 22:58:32,974: topic[/local_control/initialpose] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 22:58:32,974: topic[/local_control/initialpose] removing connection to /local_control_initialpose
[rospy.internal][INFO] 2020-10-29 22:58:32,975: topic[/remote_control] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 22:58:32,975: topic[/remote_control] removing connection to /remote_control
[rospy.internal][INFO] 2020-10-29 22:58:32,975: topic[/local_control/cancel] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 22:58:32,975: topic[/local_control/cancel] removing connection to /local_control_navigation
[rospy.internal][ERROR] 2020-10-29 22:58:32,975: Unable to initiate TCP/IP socket to freight1207:51829 (http://freight1207:58342/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 22:58:32,976: Unable to initiate TCP/IP socket to freight1207:51829 (http://freight1207:58342/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 22:58:32,976: Unable to initiate TCP/IP socket to freight1207:51829 (http://freight1207:58342/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 22:58:32,976: topic[/current/pose] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-29 22:58:32,976: Unable to initiate TCP/IP socket to freight1207:51829 (http://freight1207:58342/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 22:58:32,976: Unable to initiate TCP/IP socket to freight1207:51829 (http://freight1207:58342/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 22:58:32,976: Unable to initiate TCP/IP socket to freight1207:51829 (http://freight1207:58342/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 22:58:32,977: topic[/enable_software_runstop] removing connection to http://freight1207:58342/
[rospy.internal][INFO] 2020-10-29 22:58:32,977: topic[/local_control/goto] removing connection to http://freight1207:58342/
[rospy.internal][INFO] 2020-10-29 22:58:32,977: topic[/local_control/initialpose] removing connection to http://freight1207:58342/
[rospy.internal][INFO] 2020-10-29 22:58:32,977: topic[/remote_control] removing connection to http://freight1207:58342/
[rospy.internal][INFO] 2020-10-29 22:58:32,977: topic[/local_control/cancel] removing connection to http://freight1207:58342/
[rospy.internal][INFO] 2020-10-29 22:58:32,977: topic[/current/pose] removing connection to http://freight1207:46457/
[rospy.internal][INFO] 2020-10-29 22:58:32,977: topic[/current/pose] removing connection to http://freight1207:58342/
[rospy.impl.masterslave][INFO] 2020-10-29 22:58:32,977: Closing
[rospy.core][INFO] 2020-10-29 22:58:33,141: signal_shutdown [atexit]
[rospy.client][INFO] 2020-10-29 23:04:40,919: init_node, name[/rostful_server], pid[7629]
[xmlrpc][INFO] 2020-10-29 23:04:40,919: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-29 23:04:40,920: Started XML-RPC server [http://freight1207:54215/]
[rospy.init][INFO] 2020-10-29 23:04:40,920: ROS Slave URI: [http://freight1207:54215/]
[rospy.impl.masterslave][INFO] 2020-10-29 23:04:40,920: _ready: http://freight1207:54215/
[xmlrpc][INFO] 2020-10-29 23:04:40,920: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-29 23:04:40,920: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-29 23:04:41,020: registered with master
[rospy.rosout][INFO] 2020-10-29 23:04:41,020: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-29 23:04:41,022: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-29 23:04:41,023: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-29 23:04:41,027: rostful_server: Delaying the start 1 seconds
[rospy.internal][INFO] 2020-10-29 23:04:41,316: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2020-10-29 23:04:42,084: rostful_server: Started server on port 8080
[rospy.internal][INFO] 2020-10-29 23:04:42,086: topic[/current/pose] adding connection to [http://freight1207:49066/], count 0
[rospy.internal][INFO] 2020-10-29 23:04:42,087: topic[/current/pose] adding connection to [http://freight1207:54215/], count 1
[rospy.internal][INFO] 2020-10-29 23:04:42,089: topic[/remote_control] adding connection to [http://freight1207:54215/], count 0
[rospy.internal][INFO] 2020-10-29 23:04:42,090: topic[/enable_software_runstop] adding connection to [http://freight1207:54215/], count 0
[rospy.internal][INFO] 2020-10-29 23:04:42,091: topic[/local_control/cancel] adding connection to [http://freight1207:54215/], count 0
[rospy.internal][INFO] 2020-10-29 23:04:42,092: topic[/remote_control] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:04:42,093: topic[/local_control/initialpose] adding connection to [http://freight1207:54215/], count 0
[rospy.internal][INFO] 2020-10-29 23:04:42,093: topic[/current/pose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:04:42,094: topic[/local_control/goto] adding connection to [http://freight1207:54215/], count 0
[rospy.internal][INFO] 2020-10-29 23:04:42,094: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:04:42,095: topic[/local_control/cancel] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:04:42,095: topic[/local_control/initialpose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:04:42,095: topic[/local_control/goto] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:04:42,145: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
[rospy.internal][INFO] 2020-10-29 23:04:42,317: topic[/local_control/initialpose] adding connection to [/local_control_initialpose], count 1
[rospy.internal][INFO] 2020-10-29 23:04:42,318: topic[/local_control/cancel] adding connection to [/local_control_navigation], count 1
[rospy.internal][INFO] 2020-10-29 23:04:42,318: topic[/local_control/goto] adding connection to [/local_control_navigation], count 1
[rospy.internal][INFO] 2020-10-29 23:04:42,318: topic[/remote_control] adding connection to [/remote_control], count 1
[rosout][WARNING] 2020-10-29 23:04:52,063: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 23:04:54,584: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 23:05:28,485: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 23:05:33,988: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 23:06:56,714: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rospy.internal][WARNING] 2020-10-29 23:07:50,218: Unknown error initiating TCP/IP socket to freight1207:58796 (http://freight1207:49066/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rosgraph/network.py", line 363, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-10-29 23:07:50,218: topic[/current/pose] removing connection to http://freight1207:49066/
[rospy.client][INFO] 2020-10-29 23:23:05,557: init_node, name[/rostful_server], pid[11297]
[xmlrpc][INFO] 2020-10-29 23:23:05,557: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-29 23:23:05,557: Started XML-RPC server [http://freight1207:41141/]
[rospy.init][INFO] 2020-10-29 23:23:05,557: ROS Slave URI: [http://freight1207:41141/]
[rospy.impl.masterslave][INFO] 2020-10-29 23:23:05,557: _ready: http://freight1207:41141/
[xmlrpc][INFO] 2020-10-29 23:23:05,558: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-29 23:23:05,558: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-29 23:23:05,658: registered with master
[rospy.rosout][INFO] 2020-10-29 23:23:05,658: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-29 23:23:05,659: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-29 23:23:05,660: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-29 23:23:05,664: rostful_server: Delaying the start 1 seconds
[rospy.impl.masterslave][INFO] 2020-10-29 23:23:05,667: external shutdown request from [/master]: new node registered with same name
[rospy.core][INFO] 2020-10-29 23:23:05,667: signal_shutdown [external shutdown request from [/master]: [new node registered with same name]]
[rospy.internal][INFO] 2020-10-29 23:23:05,671: topic[/enable_software_runstop] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:23:05,671: topic[/enable_software_runstop] removing connection to /software_runstop
[rospy.internal][INFO] 2020-10-29 23:23:05,671: topic[/local_control/goto] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-29 23:23:05,672: Unable to initiate TCP/IP socket to freight1207:58468 (http://freight1207:54215/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:23:05,672: topic[/local_control/goto] removing connection to /local_control_navigation
[rospy.internal][INFO] 2020-10-29 23:23:05,672: topic[/rosout] removing connection to /rosout
[rospy.internal][ERROR] 2020-10-29 23:23:05,672: Unable to initiate TCP/IP socket to freight1207:58468 (http://freight1207:54215/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:23:05,672: topic[/local_control/initialpose] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:23:05,672: topic[/local_control/initialpose] removing connection to /local_control_initialpose
[rospy.internal][INFO] 2020-10-29 23:23:05,673: topic[/remote_control] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-29 23:23:05,673: Unable to initiate TCP/IP socket to freight1207:58468 (http://freight1207:54215/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:23:05,673: topic[/remote_control] removing connection to /remote_control
[rospy.internal][ERROR] 2020-10-29 23:23:05,673: Unable to initiate TCP/IP socket to freight1207:58468 (http://freight1207:54215/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:23:05,673: topic[/local_control/cancel] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:23:05,673: topic[/local_control/cancel] removing connection to /local_control_navigation
[rospy.internal][INFO] 2020-10-29 23:23:05,674: topic[/current/pose] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:23:05,674: topic[/enable_software_runstop] removing connection to http://freight1207:54215/
[rospy.internal][ERROR] 2020-10-29 23:23:05,674: Unable to initiate TCP/IP socket to freight1207:58468 (http://freight1207:54215/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:23:05,674: topic[/local_control/goto] removing connection to http://freight1207:54215/
[rospy.internal][ERROR] 2020-10-29 23:23:05,674: Unable to initiate TCP/IP socket to freight1207:58468 (http://freight1207:54215/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:23:05,674: topic[/local_control/initialpose] removing connection to http://freight1207:54215/
[rospy.internal][INFO] 2020-10-29 23:23:05,674: topic[/remote_control] removing connection to http://freight1207:54215/
[rospy.internal][INFO] 2020-10-29 23:23:05,675: topic[/local_control/cancel] removing connection to http://freight1207:54215/
[rospy.internal][INFO] 2020-10-29 23:23:05,675: topic[/current/pose] removing connection to http://freight1207:54215/
[rospy.internal][INFO] 2020-10-29 23:23:05,869: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,723: topic[/current/pose] adding connection to [http://freight1207:41141/], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,725: topic[/remote_control] adding connection to [http://freight1207:41141/], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,726: topic[/enable_software_runstop] adding connection to [http://freight1207:41141/], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,727: topic[/remote_control] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,728: topic[/local_control/cancel] adding connection to [http://freight1207:41141/], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,728: topic[/current/pose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,729: topic[/local_control/initialpose] adding connection to [http://freight1207:41141/], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,730: topic[/local_control/goto] adding connection to [http://freight1207:41141/], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,730: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,730: topic[/local_control/cancel] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,731: topic[/local_control/goto] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,731: topic[/local_control/initialpose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:23:06,737: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
[rospy.core][INFO] 2020-10-29 23:23:06,810: signal_shutdown [atexit]
[rospy.internal][INFO] 2020-10-29 23:23:06,814: topic[/enable_software_runstop] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:23:06,815: topic[/enable_software_runstop] removing connection to /software_runstop
[rospy.internal][INFO] 2020-10-29 23:23:06,815: topic[/local_control/goto] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:23:06,815: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-10-29 23:23:06,815: topic[/local_control/initialpose] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:23:06,816: topic[/remote_control] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-29 23:23:06,816: Unable to initiate TCP/IP socket to freight1207:37201 (http://freight1207:41141/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 23:23:06,816: Unable to initiate TCP/IP socket to freight1207:37201 (http://freight1207:41141/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:23:06,817: topic[/local_control/cancel] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-29 23:23:06,817: Unable to initiate TCP/IP socket to freight1207:37201 (http://freight1207:41141/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 23:23:06,817: Unable to initiate TCP/IP socket to freight1207:37201 (http://freight1207:41141/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 23:23:06,817: Unable to initiate TCP/IP socket to freight1207:37201 (http://freight1207:41141/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:23:06,817: topic[/current/pose] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:23:06,817: topic[/enable_software_runstop] removing connection to http://freight1207:41141/
[rospy.internal][ERROR] 2020-10-29 23:23:06,818: Unable to initiate TCP/IP socket to freight1207:37201 (http://freight1207:41141/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:23:06,818: topic[/local_control/goto] removing connection to http://freight1207:41141/
[rospy.internal][INFO] 2020-10-29 23:23:06,818: topic[/local_control/initialpose] removing connection to http://freight1207:41141/
[rospy.internal][INFO] 2020-10-29 23:23:06,818: topic[/remote_control] removing connection to http://freight1207:41141/
[rospy.internal][INFO] 2020-10-29 23:23:06,818: topic[/local_control/cancel] removing connection to http://freight1207:41141/
[rospy.internal][INFO] 2020-10-29 23:23:06,818: topic[/current/pose] removing connection to http://freight1207:41141/
[rospy.impl.masterslave][INFO] 2020-10-29 23:23:06,818: atexit
[rospy.client][INFO] 2020-10-29 23:23:45,714: init_node, name[/rostful_server], pid[11882]
[xmlrpc][INFO] 2020-10-29 23:23:45,714: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-29 23:23:45,715: Started XML-RPC server [http://freight1207:43364/]
[rospy.init][INFO] 2020-10-29 23:23:45,715: ROS Slave URI: [http://freight1207:43364/]
[rospy.impl.masterslave][INFO] 2020-10-29 23:23:45,715: _ready: http://freight1207:43364/
[xmlrpc][INFO] 2020-10-29 23:23:45,715: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-29 23:23:45,715: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-29 23:23:45,815: registered with master
[rospy.rosout][INFO] 2020-10-29 23:23:45,815: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-29 23:23:45,817: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-29 23:23:45,817: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-29 23:23:45,821: rostful_server: Delaying the start 1 seconds
[rospy.internal][INFO] 2020-10-29 23:23:46,025: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,881: topic[/current/pose] adding connection to [http://freight1207:43364/], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,883: topic[/enable_software_runstop] adding connection to [http://freight1207:43364/], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,884: topic[/remote_control] adding connection to [http://freight1207:43364/], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,885: topic[/current/pose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,885: topic[/local_control/initialpose] adding connection to [http://freight1207:43364/], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,886: topic[/remote_control] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,887: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,887: topic[/local_control/cancel] adding connection to [http://freight1207:43364/], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,888: topic[/local_control/goto] adding connection to [http://freight1207:43364/], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,888: topic[/local_control/initialpose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,888: topic[/local_control/cancel] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,889: topic[/local_control/goto] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:23:46,909: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
[rospy.core][INFO] 2020-10-29 23:23:46,966: signal_shutdown [atexit]
[rospy.internal][INFO] 2020-10-29 23:23:46,969: topic[/enable_software_runstop] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:23:46,969: topic[/enable_software_runstop] removing connection to /software_runstop
[rospy.internal][INFO] 2020-10-29 23:23:46,970: topic[/local_control/goto] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:23:46,970: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-10-29 23:23:46,970: topic[/local_control/initialpose] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:23:46,971: topic[/remote_control] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:23:46,971: topic[/local_control/cancel] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-29 23:23:46,971: Unable to initiate TCP/IP socket to freight1207:35358 (http://freight1207:43364/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 23:23:46,971: Unable to initiate TCP/IP socket to freight1207:35358 (http://freight1207:43364/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:23:46,971: topic[/current/pose] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-29 23:23:46,972: Unable to initiate TCP/IP socket to freight1207:35358 (http://freight1207:43364/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 23:23:46,972: Unable to initiate TCP/IP socket to freight1207:35358 (http://freight1207:43364/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 23:23:46,972: Unable to initiate TCP/IP socket to freight1207:35358 (http://freight1207:43364/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:23:46,972: topic[/enable_software_runstop] removing connection to http://freight1207:43364/
[rospy.internal][ERROR] 2020-10-29 23:23:46,972: Unable to initiate TCP/IP socket to freight1207:35358 (http://freight1207:43364/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:23:46,972: topic[/local_control/goto] removing connection to http://freight1207:43364/
[rospy.internal][INFO] 2020-10-29 23:23:46,973: topic[/local_control/initialpose] removing connection to http://freight1207:43364/
[rospy.internal][INFO] 2020-10-29 23:23:46,973: topic[/remote_control] removing connection to http://freight1207:43364/
[rospy.internal][INFO] 2020-10-29 23:23:46,973: topic[/local_control/cancel] removing connection to http://freight1207:43364/
[rospy.internal][INFO] 2020-10-29 23:23:46,973: topic[/current/pose] removing connection to http://freight1207:43364/
[rospy.impl.masterslave][INFO] 2020-10-29 23:23:46,973: atexit
[rospy.client][INFO] 2020-10-29 23:24:12,481: init_node, name[/rostful_server], pid[12121]
[xmlrpc][INFO] 2020-10-29 23:24:12,481: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-29 23:24:12,481: Started XML-RPC server [http://freight1207:44709/]
[rospy.init][INFO] 2020-10-29 23:24:12,481: ROS Slave URI: [http://freight1207:44709/]
[rospy.impl.masterslave][INFO] 2020-10-29 23:24:12,481: _ready: http://freight1207:44709/
[xmlrpc][INFO] 2020-10-29 23:24:12,481: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-29 23:24:12,482: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-29 23:24:12,581: registered with master
[rospy.rosout][INFO] 2020-10-29 23:24:12,582: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-29 23:24:12,583: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-29 23:24:12,584: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-29 23:24:12,588: rostful_server: Delaying the start 1 seconds
[rospy.core][INFO] 2020-10-29 23:24:12,748: signal_shutdown [atexit]
[rospy.impl.masterslave][INFO] 2020-10-29 23:24:12,750: atexit
[rosout][INFO] 2020-10-29 23:24:27,505: rostful_server: Shutting down the server
[rospy.core][INFO] 2020-10-29 23:24:27,506: signal_shutdown [Closing]
[rospy.core][INFO] 2020-10-29 23:24:27,506: signal_shutdown [atexit]
[rospy.client][INFO] 2020-10-29 23:24:32,141: init_node, name[/rostful_server], pid[12323]
[xmlrpc][INFO] 2020-10-29 23:24:32,141: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-29 23:24:32,141: Started XML-RPC server [http://freight1207:59910/]
[rospy.init][INFO] 2020-10-29 23:24:32,141: ROS Slave URI: [http://freight1207:59910/]
[rospy.impl.masterslave][INFO] 2020-10-29 23:24:32,141: _ready: http://freight1207:59910/
[xmlrpc][INFO] 2020-10-29 23:24:32,142: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-29 23:24:32,142: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-29 23:24:32,242: registered with master
[rospy.rosout][INFO] 2020-10-29 23:24:32,242: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-29 23:24:32,243: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-29 23:24:32,244: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-29 23:24:32,248: rostful_server: Delaying the start 1 seconds
[rospy.internal][INFO] 2020-10-29 23:24:32,494: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2020-10-29 23:24:33,302: rostful_server: Started server on port 8080
[rospy.internal][INFO] 2020-10-29 23:24:33,308: topic[/current/pose] adding connection to [http://freight1207:59910/], count 0
[rospy.internal][INFO] 2020-10-29 23:24:33,310: topic[/enable_software_runstop] adding connection to [http://freight1207:59910/], count 0
[rospy.internal][INFO] 2020-10-29 23:24:33,310: topic[/remote_control] adding connection to [http://freight1207:59910/], count 0
[rospy.internal][INFO] 2020-10-29 23:24:33,311: topic[/local_control/cancel] adding connection to [http://freight1207:59910/], count 0
[rospy.internal][INFO] 2020-10-29 23:24:33,312: topic[/current/pose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:24:33,312: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:24:33,313: topic[/local_control/initialpose] adding connection to [http://freight1207:59910/], count 0
[rospy.internal][INFO] 2020-10-29 23:24:33,314: topic[/remote_control] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:24:33,314: topic[/local_control/goto] adding connection to [http://freight1207:59910/], count 0
[rospy.internal][INFO] 2020-10-29 23:24:33,315: topic[/local_control/cancel] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:24:33,315: topic[/local_control/initialpose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:24:33,315: topic[/local_control/goto] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:24:33,334: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
[rosout][WARNING] 2020-10-29 23:24:39,337: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 23:24:46,828: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][INFO] 2020-10-29 23:28:35,711: rostful_server: Shutting down the server
[rospy.core][INFO] 2020-10-29 23:28:35,711: signal_shutdown [Closing]
[rospy.internal][INFO] 2020-10-29 23:28:35,715: topic[/enable_software_runstop] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:28:35,715: topic[/enable_software_runstop] removing connection to /software_runstop
[rospy.internal][INFO] 2020-10-29 23:28:35,716: topic[/local_control/goto] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:28:35,716: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-10-29 23:28:35,716: topic[/local_control/initialpose] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:28:35,717: topic[/remote_control] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-29 23:28:35,717: Unable to initiate TCP/IP socket to freight1207:37764 (http://freight1207:59910/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 23:28:35,717: Unable to initiate TCP/IP socket to freight1207:37764 (http://freight1207:59910/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 23:28:35,717: Unable to initiate TCP/IP socket to freight1207:37764 (http://freight1207:59910/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:28:35,717: topic[/local_control/cancel] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-29 23:28:35,718: Unable to initiate TCP/IP socket to freight1207:37764 (http://freight1207:59910/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-29 23:28:35,718: Unable to initiate TCP/IP socket to freight1207:37764 (http://freight1207:59910/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:28:35,718: topic[/current/pose] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-29 23:28:35,718: topic[/enable_software_runstop] removing connection to http://freight1207:59910/
[rospy.internal][ERROR] 2020-10-29 23:28:35,718: Unable to initiate TCP/IP socket to freight1207:37764 (http://freight1207:59910/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-29 23:28:35,718: topic[/local_control/goto] removing connection to http://freight1207:59910/
[rospy.internal][INFO] 2020-10-29 23:28:35,718: topic[/local_control/initialpose] removing connection to http://freight1207:59910/
[rospy.internal][INFO] 2020-10-29 23:28:35,718: topic[/remote_control] removing connection to http://freight1207:59910/
[rospy.internal][INFO] 2020-10-29 23:28:35,719: topic[/local_control/cancel] removing connection to http://freight1207:59910/
[rospy.internal][INFO] 2020-10-29 23:28:35,719: topic[/current/pose] removing connection to http://freight1207:59910/
[rospy.impl.masterslave][INFO] 2020-10-29 23:28:35,719: Closing
[rospy.core][INFO] 2020-10-29 23:28:35,834: signal_shutdown [atexit]
[rospy.client][INFO] 2020-10-29 23:28:38,910: init_node, name[/rostful_server], pid[13125]
[xmlrpc][INFO] 2020-10-29 23:28:38,910: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-29 23:28:38,911: Started XML-RPC server [http://freight1207:59440/]
[rospy.init][INFO] 2020-10-29 23:28:38,911: ROS Slave URI: [http://freight1207:59440/]
[rospy.impl.masterslave][INFO] 2020-10-29 23:28:38,911: _ready: http://freight1207:59440/
[xmlrpc][INFO] 2020-10-29 23:28:38,911: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-29 23:28:38,911: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-29 23:28:39,011: registered with master
[rospy.rosout][INFO] 2020-10-29 23:28:39,011: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-29 23:28:39,013: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-29 23:28:39,013: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-29 23:28:39,018: rostful_server: Delaying the start 1 seconds
[rospy.internal][INFO] 2020-10-29 23:28:39,242: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2020-10-29 23:28:40,067: rostful_server: Started server on port 8080
[rospy.internal][INFO] 2020-10-29 23:28:40,077: topic[/current/pose] adding connection to [http://freight1207:59440/], count 0
[rospy.internal][INFO] 2020-10-29 23:28:40,079: topic[/enable_software_runstop] adding connection to [http://freight1207:59440/], count 0
[rospy.internal][INFO] 2020-10-29 23:28:40,080: topic[/remote_control] adding connection to [http://freight1207:59440/], count 0
[rospy.internal][INFO] 2020-10-29 23:28:40,081: topic[/local_control/cancel] adding connection to [http://freight1207:59440/], count 0
[rospy.internal][INFO] 2020-10-29 23:28:40,081: topic[/current/pose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:28:40,082: topic[/local_control/initialpose] adding connection to [http://freight1207:59440/], count 0
[rospy.internal][INFO] 2020-10-29 23:28:40,083: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:28:40,084: topic[/local_control/goto] adding connection to [http://freight1207:59440/], count 0
[rospy.internal][INFO] 2020-10-29 23:28:40,084: topic[/local_control/cancel] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:28:40,084: topic[/remote_control] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:28:40,084: topic[/local_control/initialpose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:28:40,085: topic[/local_control/goto] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-29 23:28:40,103: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
[rospy.internal][INFO] 2020-10-29 23:28:56,999: topic[/current/pose] adding connection to [http://freight1207:52045/], count 1
[rospy.internal][INFO] 2020-10-29 23:28:57,168: topic[/local_control/goto] adding connection to [/local_control_navigation], count 1
[rospy.internal][INFO] 2020-10-29 23:28:57,168: topic[/local_control/initialpose] adding connection to [/local_control_initialpose], count 1
[rospy.internal][INFO] 2020-10-29 23:28:57,169: topic[/local_control/cancel] adding connection to [/local_control_navigation], count 1
[rospy.internal][INFO] 2020-10-29 23:29:09,384: topic[/remote_control] adding connection to [/remote_control], count 1
[rosout][WARNING] 2020-10-29 23:29:12,945: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rosout][WARNING] 2020-10-29 23:30:01,252: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rospy.internal][WARNING] 2020-10-29 23:30:06,560: Unknown error initiating TCP/IP socket to freight1207:34461 (http://freight1207:52045/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rosgraph/network.py", line 363, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-10-29 23:30:06,560: topic[/current/pose] removing connection to http://freight1207:52045/
[rosout][WARNING] 2020-10-29 23:30:14,584: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rospy.internal][INFO] 2020-10-29 23:30:22,083: topic[/current/pose] adding connection to [http://freight1207:58622/], count 1
[rospy.internal][INFO] 2020-10-29 23:30:22,283: topic[/local_control/initialpose] adding connection to [/local_control_initialpose], count 2
[rospy.internal][INFO] 2020-10-29 23:30:22,283: topic[/local_control/initialpose] removing connection to /local_control_initialpose
[rospy.internal][INFO] 2020-10-29 23:30:22,284: topic[/local_control/goto] adding connection to [/local_control_navigation], count 2
[rospy.internal][INFO] 2020-10-29 23:30:22,284: topic[/local_control/goto] removing connection to /local_control_navigation
[rospy.internal][INFO] 2020-10-29 23:30:22,284: topic[/local_control/cancel] adding connection to [/local_control_navigation], count 2
[rospy.internal][INFO] 2020-10-29 23:30:22,284: topic[/local_control/cancel] removing connection to /local_control_navigation
[rosout][WARNING] 2020-10-29 23:30:26,135: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: -0.87
    locationY: 0.48
    orientation: 0.9
  - 
    locationX: -0.01
    locationY: 2.41
    orientation: 0.9 to topic /local_control/goto
[rospy.internal][WARNING] 2020-10-29 23:33:44,829: Unknown error initiating TCP/IP socket to freight1207:43873 (http://freight1207:58622/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rosgraph/network.py", line 363, in read_ros_handshake_header
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2020-10-29 23:33:44,829: topic[/current/pose] removing connection to http://freight1207:58622/
[rospy.internal][INFO] 2020-10-29 23:34:42,131: topic[/enable_software_runstop] adding connection to [/software_runstop], count 2
[rospy.internal][INFO] 2020-10-29 23:34:42,132: topic[/enable_software_runstop] removing connection to /software_runstop
